e2e/profiles/profiles-both-profiles-no-settings.e2e.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright © 2023 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { test } from '@playwright/test';
+
+import {
+  clearSettings,
+  clearUserProfile,
+  testForNoFirstRunWindow,
+  verifySystemProfile,
+  verifyUserProfile,
+} from '../utils/ProfileUtils';
+
+test.describe.serial('sys-profile with settings', () => {
```

Comment:
To me, it doesn't make sense to have each test case in its own file. It would be cleaner to have it all in a single file at `e2e/profiles.e2e.spec.ts`. Giving each test case its own file results in a lot of boilerplate, and doesn't have any benefits - you can still filter tests if they are all in one file. I could be missing something though...

Reply:
Did that, and it more or less works out. The first 8 tests are now in a table, for (yes-settings, no-settings) x (yes-user-profile, no-user-profile) x (yes-system-profile, no-system-profile) . Then there are 3 other tests for the 3 specific problems in profiles (bad types, unrecognized settings, and bad structure).